core/dbt/artifacts/resources/v1/metric.py

```diff
@@ -95,7 +103,9 @@ class MetricTypeParams(dbtClassMixin):
     denominator: Optional[MetricInput] = None
     expr: Optional[str] = None
     window: Optional[MetricTimeWindow] = None
-    grain_to_date: Optional[TimeGranularity] = None
+    grain_to_date: Optional[TimeGranularity] = (
```

Comment:
✅ I sanity checked that this one _didn't change_ in the manifest v12.json. Happy to report that it correctly stayed the same 🙂